pypesto/objective/amici_calculator.py

```diff
@@ -72,6 +73,8 @@ def __call__(self,
         fim_for_hess:
             Whether to use the FIM (if available) instead of the Hessian (if
             requested).
+        chunk_size:
+            Size of chunks the ExpData is sliced into.
```

Comment:
I would try to use the docstring from the current amici_predictor, which is I think more exhaustive. The point is that this chunk_size feature may be very important for certain applications, but it's not necessarily intuitive, especially if you're not familiar with AMICI...